pax_express_client/cli.py

```diff
@@ -7,7 +8,24 @@
     file_cli,
     version_cli,
 )
-
+import keyring
+from keyring.backends import Windows, OS_X
```

Comment:
wouldn't it be better to only import those on the respective systems or does that break nuitka?